scripts/migration_assistant/generate_script.py

```diff
@@ -30,16 +31,15 @@ def get_ontology_term_map(term_map_filepath) -> dict:
         return replaced_by_map
 
 
-def get_genecode_term_map() -> List[str]:
-    # TODO: read in the gencode term map
-    return []
+def migrate_gencode() -> bool:
+    return len(get_deprecated_feature_ids()) > 1
```

Comment:
I made an [issue to track this separately](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-data-portal/4988), but one note--this will always return True because we never re-set the diff.txt's back to blank after a migration. So the repo will retain the list of the previous diff set even if the next migration doesn't update the GENCODE files. Probably not in scope to fix here, unless you think there's an easy place to resolve that as part of this PR